### PR TITLE
Fledge-Pkg changes for postgres13 upgrade(FOGL-6348)

### PR DESCRIPTION
**Pull request by Innovative-ashwin**
Signed-off-by: Innovative-ashwin <sinha.ashwini111@gmail.com>

### CODE REVIEW

File: make_rpm [hunk 1]
```diff
@@ -124,15 +124,15 @@ if [[ $skip_build == 0 ]]; then
 
 	# Enables/verifies the environment for the build
 	source scl_source enable ${rhpg_pkg}
-	status_rh_postgresql96=$?
+	status_rh_postgresql13=$?
```
Comment (YashTatkondawar) [inline make_rpm:127]: Please rename `status_rh_postgresql13` to `status_rh_postgresql`, so that it becomes generic.

File: make_rpm [hunk 2]
```diff
@@ -124,15 +124,15 @@ if [[ $skip_build == 0 ]]; then
...
 
 	source scl_source enable ${rhgcc_pkg}
 	status_devtoolset=$?
 
 	command -v pg_isready > /dev/null
 	status_pg_isready=$?
 
-	if [[ $status_rh_postgresql96 != 0 ]]; then
+	if [[ $status_rh_postgresql13 != 0 ]]; then
```
Comment (YashTatkondawar) [inline make_rpm:135]: Please rename `status_rh_postgresql13` to `status_rh_postgresql`, so that it becomes generic.

File: make_rpm (hunk 1 above)
Comment (Innovative-ashwin) [reply make_rpm:127]: fixed

**Review by YashTatkondawar — APPROVED** [review]

File: make_rpm (hunk 2 above)
Comment (Innovative-ashwin) [reply make_rpm:135]: fixed

**Review by YashTatkondawar — APPROVED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**